<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>795</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>795</width>
    <height>600</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>795</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="radiusSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="altitudeSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="swathSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="invertedControls">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="rectifyButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Rectify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>520</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="rectifyProgress">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>520</y>
      <width>351</width>
      <height>23</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="earthRadiusLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Earth Radius</string>
    </property>
   </widget>
   <widget class="QLabel" name="altitudeLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>270</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Satellite Altitude</string>
    </property>
   </widget>
   <widget class="QLabel" name="swathLabel">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Satellite Swath</string>
    </property>
   </widget>
   <widget class="QPushButton" name="openButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Open</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="openLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="saveLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>291</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sliderResetButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>381</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="earthSliderDisplay">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>230</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="altitudeSliderDisplay">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="swathSliderDisplay">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>64</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="segmentStyle">
     <enum>QLCDNumber::Flat</enum>
    </property>
   </widget>
   <widget class="QLabel" name="imageView">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>351</width>
      <height>481</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextBrowser" name="logBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>120</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="HighlightedText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>120</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="HighlightedText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Highlight">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>120</green>
          <blue>215</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="HighlightedText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>795</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>openButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>162</x>
     <y>208</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radiusSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>earthSliderDisplay</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>84</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>altitudeSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>altitudeSliderDisplay</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>300</y>
    </hint>
    <hint type="destinationlabel">
     <x>100</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>swathSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>swathSliderDisplay</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radiusSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateSlider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>239</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>altitudeSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateSlider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>301</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>swathSlider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateSlider()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>355</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>383</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderResetButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>resetClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>321</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>414</x>
     <y>381</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rectifyButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>rectifyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>347</x>
     <y>503</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>481</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>560</y>
    </hint>
    <hint type="destinationlabel">
     <x>104</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>saveLineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>updateSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>365</x>
     <y>567</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MainWindow</sender>
   <signal>setProgressValue(int)</signal>
   <receiver>rectifyProgress</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>528</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>setProgressValue(int)</signal>
  <slot>openClicked()</slot>
  <slot>updateSlider()</slot>
  <slot>resetClicked()</slot>
  <slot>rectifyClicked()</slot>
  <slot>saveClicked()</slot>
  <slot>updateOpen()</slot>
  <slot>updateSave()</slot>
 </slots>
</ui>
